#Scenes/Map31B5.tmp

```diff
@@ -0,0 +1,89 @@
+[gd_scene load_steps=4 format=3 uid="uid://ki060lfrgty4"]
```

Comment:
Can we remove this .tmp file? This looks to just be MapUI.tscn

Reply:
mmh I'm not sure, i'll remove it and see if nothing breaks